### DISCUSSION TITLE
How to solve the following problems in matlab

### DISCUSSION THREAD

**Discussion by RunfangHouSCAU** [community/community#48559] [Programming Help]
Output argument "versionGit" (and possibly others) not assigned a value in the execution with
"initCobraToolbox>checkGit" function.

出错 initCobraToolbox (第 122 行)
[installedGit, versionGit] = checkGit();

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @RunfangHouSCAU ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
